app/frontend/src/registry/components/people/ApplicationAdd.vue

```diff
@@ -1,218 +1,156 @@
+          </b-row>
+        </b-form-group>
+        <b-row>
+          <b-col md="2">Select classification</b-col>
+          <b-col md="8">
+            <b-form-radio-group class="fixed-width" :options="formOptions.classifications" @change="changedClassification" v-model="qualificationForm.subactivity" required></b-form-radio-group>
+          </b-col>
+        </b-row>
+        <b-form-group label="Qualified to drill">
+          <b-row>
+            <b-col md="8">
+              <b-form-checkbox-group class="fixed-width" :options="formOptions.qualifications" v-model="qualificationForm.qualifications" disabled>
+              </b-form-checkbox-group>
+            </b-col>
+          </b-row>
+        </b-form-group>
+        <b-form-group label="Date application received">
+          <b-row>
+            <b-col>
+              <datepicker format="yyyy-MM-dd" v-model="qualificationForm.status_set[0].effective_date" required></datepicker>
+            </b-col>
+          </b-row>
+        </b-form-group>
+      </p>
     </div>
-    <fieldset class="registry-section">
-      <legend>Adjudication</legend>
-      <form class="form-horizontal">
-        <div class="form-group">
-          <label class="col-xs-12 col-md-1" for="dateAppReceived">Date application received:</label>
-          <div class="col-xs-12 col-md-3 form-spacing">
-            <input type="text" id="dateAppReceived" placeholder="Select date" v-model="editClassification.application.receivedDate">
-          </div>
-        </div>
-        <div class="form-group">
-          <label class="col-xs-12 col-md-1" for="approvalOutcomeDate">Approval outcome date:</label>
-          <div class="col-xs-12 col-md-3 form-spacing">
-            <input type="text" id="approvalOutcomeDate" placeholder="Select date" v-model="editClassification.application.approvalOutcomeDate">
-          </div>
-          <label class="col-xs-12 col-md-1" for="approvalOutcome">Approval outcome:</label>
-          <div class="col-xs-12 col-md-3 form-spacing">
-            <input type="text" id="approvalOutcome" placeholder="Enter approval outcome" v-model="editClassification.application.approvalOutcome">
-          </div>
-            <label class="col-xs-12 col-md-1" for="reasonDenied">Reason not approved:</label>
-            <div class="col-xs-12 col-md-3 form-spacing">
-              <input type="text" id="reasonDenied" placeholder="Enter reason not approved" v-model="editClassification.application.reasonDenied">
-            </div>
-        </div>
-        <div class="form-group">
-          <label class="col-xs-12 col-md-1" for="removalDate">Register removal date:</label>
-          <div class="col-xs-12 col-md-3 form-spacing">
-            <input type="text" id="removalDate" placeholder="Select removal date" v-model="editClassification.application.removalDate">
-          </div>
-        </div>
-      </form>
-    </fieldset>
   </div>
 </template>
 
 <script>
+import Datepicker from 'vuejs-datepicker'
```

Comment:
I don't think we need to use this - can just use the std. html date picker: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

Reply:
per MDN, the html datepicker is not supported by IE or Safari, and unfortunately some users might need IE11. We should try out our options in IE11

Reply:
Sticking with the datepicker then.